### PR TITLE
Merge master feature/release2 wa2

### PR DESCRIPTION
**Pull request by udaydenduluri33**
Getting the latest of master onto feature/release2-wa2 by resolving the conflicts

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
As this is a merge of existing code on master I can approve

File: src/style/app.scss [hunk 1]
```diff
@@ -388,3 +388,8 @@ button:focus {
     background-color: #ffdd00 !important;
   }
 }
+
+exui-task-list {
```
Comment (kuda1992) [inline src/style/app.scss:392]: We should remove this not needed

**Review by kuda1992 — CHANGES_REQUESTED** [review]
Looks fine apart from the code I mentioned to be removed.

**Review by rayliangatos — APPROVED** [review]

**Review by sreekanthpuligadda — APPROVED** [review]
looks good

**Pull request merged**